tdvt/tdvt/setup_env.py

```diff
@@ -81,14 +87,21 @@ def create_ds_ini_file(name, logical_config):
         ini.write('\n')
         ini.write('[UnionTest]\n')
         ini.write('\n')
+        ini.write('[StaplesDataTest]\n')
```

Comment:
I don't know if we want to include this by default. It's a bit problematic since it doesn't support column renaming and the staples table has spaces and capitalization which some databases don't support.

Reply:
Good point, I've removed it.